src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    // Indicates that the modified instance is pinned in memory.
+    public static class IsPinned 
+    {
+    }
```

Comment:
Can you please move *all* these marker classes to this file (IsConst, IsByValue, etc.)? I does not make sense to have some in coreclr and some other in corefx.